web/src/components/vis/AnovaTable.vue

```diff
@@ -32,33 +31,23 @@ export default {
         },
         {
           text: 'Intercept',
-          align: 'right',
```

Comment:
Why remove this alignment?  Column headers should be aligned with data.

Reply:
I don't know any spreadsheet application that automatically aligns column labels to the right if the values are numbers

Reply:
It only caught my eye because headers are closer to their neighbor column than their own.  The ui here also doesn't really make me think "spreadsheet" like the cleanup table does.

For what it's worth, I believe spreadsheets align numbers to the right to be consistent with accounting software and books, so that convention may or may not be important to a user (I don't really know).

Reply:
well they do it cause it easier to compare numbers that way:
```
23123.23
  232.23
```

vs

```
23123.23
232.23
```

Reply:
No, it's not a blocker.